src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -76,7 +78,7 @@ static int uct_rdmacm_is_addr_route_resolved(struct rdma_cm_id *cm_id,
 {
     char ip_port_str[UCS_SOCKADDR_STRING_LEN];
     ucs_status_t status;
-    int event_type;
+    enum rdma_cm_event_type event_type;
```

Comment:
move this var declaration to before line 80

Reply:
moved